relay-server/src/actors/healthcheck.rs

```diff
@@ -59,6 +59,9 @@ impl Handler<IsHealthy> for Healthcheck {
     type Result = ResponseFuture<bool, ()>;
 
     fn handle(&mut self, message: IsHealthy, _context: &mut Self::Context) -> Self::Result {
+        if self.config.relay_mode() == RelayMode::Managed {
+            self.upstream.do_send(IsNetworkOutage);
```

Comment:
Please clarify in a comment that we only care about the metric here, not about the result.

_nit_: Add an empty line after the `if` block.

Reply:
Moved the metric emitting code here.